apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -502,6 +555,70 @@ export class DotEditContentHtmlService {
         );
     }
 
+    private resetInlineCurrentContent(id: string) {
+        return this.inlineCurrentContent[id];
+    }
+
+    private handleDatasetMissingErrors({ dataset }: Pick<DotInlineEditContent, 'dataset'>): void {
+        const requiredDatasetKeys = ['mode', 'inode', 'fieldName', 'language'];
+        const datasetMissing = requiredDatasetKeys.filter(function (key) {
+            return !Object.keys(dataset).includes(key);
+        });
+        const message = this.dotMessageService.get('editpage.inline.attribute.error');
+        datasetMissing.forEach((dataset) => {
+            this.dotGlobalMessageService.error(`${dataset} ${message}`);
+        });
+    }
+
+    private handleTinyMCEOnFocusEvent(contentlet: DotInlineEditContent) {
+        this.handleDatasetMissingErrors(contentlet);
+        this.inlineCurrentContent = {
+            ...this.inlineCurrentContent,
+            [contentlet.element.id]: contentlet.innerHTML
+        };        
+    }
+
+    private handleTinyMCEOnBlurEvent(content: DotInlineEditContent) {
+
+        // If editor is dirty then we continue making the request
+        if (!content.isNotDirty) {
+            // Add the loading indicator to the field
+            content.element.classList.add('inline-editing--saving');
+
+            const contentTypeNode: HTMLElement = content.element.closest(
+                '[data-dot-object="contentlet"]'
+            );
+
+            const { dotType } = contentTypeNode.dataset;  
+
+            // All good, initiate the request
+            this.dotWorkflowActionsFireService
+                .saveContentlet(dotType, {
+                    [content.dataset.fieldName]: content.innerHTML,
+                    inode: content.dataset.inode
+                })
+                .pipe(take(1))
+                .subscribe(
+                    () => {
+                        // on success
```

Comment:
nothing happens in success ?

Reply:
not sure if the process that saves sends the msg to the UI of saved
![image](https://user-images.githubusercontent.com/37185433/114434590-6ce50680-9b80-11eb-9ca2-58f087a586aa.png)

Reply:
@hmoreras @alfredo-dotcms Yes, nothing happens in success.